internal/infrastructure/reservation/postgresql/sqlc/reservations.go

```diff
@@ -2,15 +2,17 @@ package sqlc
 
 import (
 	"context"
-	"go.uber.org/zap"
 	"spot-assistant/internal/core/dto/guild"
 	"spot-assistant/internal/core/dto/member"
 	"time"
 
-	"github.com/jackc/pgx/v5"
-	"github.com/jackc/pgx/v5/pgtype"
+	"go.uber.org/zap"
+
 	"spot-assistant/internal/common/errors"
 	"spot-assistant/internal/core/dto/reservation"
+
+	"github.com/jackc/pgx/v5"
+	"github.com/jackc/pgx/v5/pgtype"
 )
```

Comment:
i'm too lazy to google now, but in general, there's this (in my opinion) perfect scheme of import formatting and grouping: 

* always follow the order of `stdlib` > `external 3rd party` > `internal` >= `internal 3rd party`
* separate these by a newline
e.g.

```go

import (
  "context"
 
  "github.com/jackc/pgx/v5"
  "github.com...."

  "spot-assistant/internal/common/errors"
)
```